os-eap-launch/added/launch/jgroups.sh

```diff
@@ -36,7 +36,8 @@ function configure_jgroups_encryption() {
       product=`read_product_name`
       version=`read_product_version`
 
-      if [ "$product" == "Red Hat JBoss Enterprise Application Platform" -a "$(version_compare $version '6.4.4.GA')" == "newer" ]; then
+      if [ "$product" == "Red Hat JBoss Enterprise Application Platform" -a "$(version_compare $version '6.4.4.GA')" == "newer" ] ||
```

Comment:
@goldmann Asked for clarification why it did work in JDG 7.0.x.

It turned out that it worked because of a bug. We didn't change the product name in `version.txt`:

```
JDG 7.0.x:
cat /opt/datagrid/version.txt
Red Hat JBoss Enterprise Application Platform - Version 7.0.4.GA

JDG 7.1.1:
cat /opt/datagrid/version.txt 
Red Hat JBoss Data Grid - Version 7.1.1
```

Reply:
I think this is OK, but I'm concerned that we've now got JDG specific code in a common EAP module.  Hindsight being 20/20, perhaps we should have had separate JGroups configuration scripts for EAP 6 and 7.

Reply:
As I mentioned in PR's description, I believe checking JGroups version might be a much better idea. I already prepared a Pull Request for this: https://github.com/jboss-openshift/cct_module/pull/164.

If you agree with me, I prefer to close this PR (as it's a defensive implementation) and pursue https://github.com/jboss-openshift/cct_module/pull/164.